scripts/html-template-brick.js

```diff
@@ -1,18 +1,61 @@
 /**
- * Simpler brick using data-aem-selector attributes in the
+ * Simpler brick using data-aem-inject attributes in the
  * HTML template to select the content to inject in it.
  */
 export default class HtmlTemplateBrick extends HTMLElement {
-  connectedCallback() {
```

Comment:
@bdelacretaz I moved this to the constructor so it's not required to add `super.connectedCallback` to each instance of this class. You can see on `arm-hero.js`